app/Models/Foundation/Summit/Events/Presentations/Presentation.php

```diff
@@ -894,6 +894,18 @@ public function setMaterials($materials)
         $this->materials = $materials;
     }
 
+    /**
+     * @param PresentationMaterial $material
+     * @return Presentation
+     */
+    public function addMaterial(PresentationMaterial $material)
```

Comment:
use this implementation here
https://github.com/OpenStackweb/summit-api/blob/ebda2ab8320efd613b70b1b7397ef31559f3c0d1/app/Models/Foundation/Summit/Events/Presentations/Presentation.php#L522
https://github.com/OpenStackweb/summit-api/blob/ebda2ab8320efd613b70b1b7397ef31559f3c0d1/app/Models/Foundation/Summit/Events/Presentations/Presentation.php#L652
https://github.com/OpenStackweb/summit-api/blob/ebda2ab8320efd613b70b1b7397ef31559f3c0d1/app/Models/Foundation/Summit/Events/Presentations/Presentation.php#L686
https://github.com/OpenStackweb/summit-api/blob/ebda2ab8320efd613b70b1b7397ef31559f3c0d1/app/Models/Foundation/Summit/Events/Presentations/Presentation.php#L735

Reply:
done